services/user.py

```diff
@@ -0,0 +1,53 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None,
+
+) -> None:
+    user = get_user_model().objects.create_user(
+        username=username,
+        password=password,
+    )
+    if email:
+        user.email = email
+    if first_name:
+        user.first_name = first_name
+    if last_name:
+        user.last_name = last_name
+
+    user.save()
+
+
+def get_user(user_id: int) -> User:
+    return get_user_model().objects.get(pk=user_id)
+
+
+def update_user(
+        user_id: int,
+        username: str = None,
+        password: str = None,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None,
+) -> None:
+    user = get_user(user_id=user_id)
```

Comment:
The `get_user` function can raise a `DoesNotExist` exception if the `User` with the given `id` is not found. Consider handling this exception to prevent potential runtime errors.